src/js/components/icons/status/OK.js

```diff
@@ -2,20 +2,33 @@
 
 var React = require('react');
 var IntlMixin = require('../../../mixins/GrommetIntlMixin');
+var uuid = require('node-uuid');
 
 var OK = React.createClass({
 
   mixins: [IntlMixin],
 
+  propTypes: {
+    a11yTitle: React.PropTypes.string
+  },
+
   render: function() {
     var className = 'status-icon status-icon-ok';
+    //var a11yTitle = this.getGrommetIntlMessage('OK');
+    var a11yTitle;
     if (this.props.className) {
       className += ' ' + this.props.className;
     }
+    if (typeof this.props.a11yTitle === "undefined") {
+      //this.props.a11yTitle = a11yTitle;
+      a11yTitle = this.getGrommetIntlMessage('OK');
```

Comment:
If the `a11yTitle` is "" then `if (!this.props.a11yTitle) {` will evaluate to true. I could see an example where the status icon is repetitive information and does not need a `title` like the Grommet logo in `Home.js`.   This would not allow the developer the option to pass in empty string to have screen readers not read the status icon.